<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cam_control</class>
 <widget class="QMainWindow" name="Cam_control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cam_control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1921</width>
      <height>1051</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="icon">
      <iconset>
       <normalon>../progbase3/build-Kursova-Desktop-Debug/Icons/tab-icon.png</normalon>
      </iconset>
     </attribute>
     <attribute name="title">
      <string>Real Time Viev</string>
     </attribute>
     <widget class="QLabel" name="label_central">
      <property name="geometry">
       <rect>
        <x>-70</x>
        <y>10</y>
        <width>960</width>
        <height>600</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../progbase3/build-Kursova-Desktop-Debug/Icons/no_image.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1280</x>
        <y>10</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../progbase3/build-Kursova-Desktop-Debug/Icons/no_image.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>10</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../progbase3/build-Kursova-Desktop-Debug/Icons/no_image.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>1280</x>
        <y>330</y>
        <width>303</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>              set camera 2 viev             </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>330</y>
        <width>303</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>              set camera 1 viev             </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>710</y>
        <width>303</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>              set camera 3 viev             </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>1280</x>
        <y>710</y>
        <width>303</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>              set camera 4 viev             </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>1280</x>
        <y>390</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../progbase3/build-Kursova-Desktop-Debug/Icons/no_image.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>390</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../progbase3/build-Kursova-Desktop-Debug/Icons/no_image.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>1600</x>
        <y>10</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../progbase3/build-Kursova-Desktop-Debug/Icons/no_image.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>1600</x>
        <y>390</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../progbase3/build-Kursova-Desktop-Debug/Icons/no_image.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>650</y>
        <width>881</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="record">
           <property name="text">
            <string> record </string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>../../../Not_progbase_stuff/Downloads/video-camera-with-play-button(1).png</normalon>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="overwatch">
           <property name="text">
            <string> overwatch </string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../Not_progbase_stuff/Downloads/eye.png</normaloff>
             <normalon>../../Not_progbase_stuff/Downloads/eye.png</normalon>../../Not_progbase_stuff/Downloads/eye.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="overwatch_policy">
           <property name="text">
            <string> overwatch policy  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Emergency">
           <property name="text">
            <string> Emergency </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>1600</x>
        <y>330</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Photo that will be saved</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>1600</x>
        <y>710</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Photo that computer see</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="icon">
      <iconset>
       <normaloff>../progbase3/build-Kursova-Desktop-Debug/Icons/alarm-red.png</normaloff>../progbase3/build-Kursova-Desktop-Debug/Icons/alarm-red.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Archives</string>
     </attribute>
     <widget class="QPushButton" name="MainScreenController">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>680</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>1500</x>
        <y>0</y>
        <width>401</width>
        <height>751</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="Folders">
         <property name="font">
          <font>
           <family>Noto Mono</family>
           <italic>false</italic>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Images</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Video</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="Entities">
         <property name="font">
          <font>
           <family>Noto Mono</family>
           <italic>false</italic>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="mainScreen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>960</width>
        <height>600</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../progbase3/build-Kursova-Desktop-Debug/photo/Thu Jun 7 2018/18:16:49.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="prev">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>10</y>
        <width>450</width>
        <height>350</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../progbase3/build-Kursova-Desktop-Debug/photo/Thu Jun 7 2018/15:38:51.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="prev_prev">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>400</y>
        <width>450</width>
        <height>350</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../progbase3/build-Kursova-Desktop-Debug/photo/Thu Jun 7 2018/15:38:51.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="prev_name">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>370</y>
        <width>451</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="prev_prev_name">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>760</y>
        <width>451</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="mainScreenName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>620</y>
        <width>961</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>MainScreen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>680</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Send data to server</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
